stephenPrintup/README.md

```diff
@@ -0,0 +1,276 @@
+### Homework #1
+(Due Monday 7/20)
+
+You need to turn in this homework by 
+
+[X]cloning this repo, 
+[X]making a new branch with your solutions, 
+[X]pushing to github, and 
+[X]issuing a pull request.
+
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_30% of total time_)
+Predict the output of each of the following expressions, and explain the rules which determine that answer.  (_Hint: The rules for operator `==` are complicated, but do your best!_)
+
+If the output depends on the value or type of variable _x_, identify the conditions (what _x_ is) when the expression will output true (or false, if that's simpler).  Assume the cases are independent, and _x_ is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
```

Comment:
Great work on adding your predictions as well as the results that came from running the code in the console. You may find it helpful to write out your reasoning even for what appears to be the simpler questions, as being able to explain why things work the way they work is an excellent way to cement knowledge. Take a look at the homework solutions in your review time, and experiment with these questions again by assigning different values to x, such as numbers or strings. Don't assume x is simply undefined!